dashboard/templates/403.html

```diff
@@ -0,0 +1 @@
+{%  extends '400.html' %}
```

Comment:
I figured that 403 forbidden might confuse some users who didn't know what a 403 was. If they don't have access then to them the link doesn't exist so it is basically 404 page not found